apps/boiler-plate/README.md

```diff
@@ -1,27 +1,129 @@
+
+The Angular CLI generates the initial file structure of your project. The root directory of your project should contain the following files and directories:
+
+src/: This directory contains all the source files of your application.
+
+angular.json: This file contains configuration options for your Angular project.
+
+package.json: This file contains dependencies and scripts for your project.
+
+tsconfig.json: This file contains TypeScript compiler options for your project.
+
+Within the src/ directory, the recommended file structure is as follows:
+
+app/: This directory contains the root component and other components, services, and modules for your application.
+
+assets/: This directory contains static assets like images, fonts, and other files that should be copied as-is to the output folder.
+
+environments/: This directory contains environment-specific configuration files.
+
+index.html: This file is the entry point for your application.
+
+main.ts: This file bootstraps your application.
+
+styles.css: This file contains global styles for your application.
+
+test.ts: This file is the entry point for your application's tests.
+
+Within the app/ directory, the recommended file structure is as follows:
+
+app.component.ts, app.component.html, and app.component.css: These files define the root component of your application.
+
+app.module.ts: This file defines the main module of your application.
+
+shared/: This directory contains shared components, services, and modules for your application.
+
+feature/: This directory contains feature-specific components, services, and modules for your application.
+
+# Project Features
+
+authComponent: this component authenticates the user and receives an access token from Auth
+
+commands:In the project we using forget-password,get-current-user,get-token,google-login,login,logout,refresh-token,reset-password,verify-reset-password commands to make the autheticated and secure appication
+
+loginComponent: The login component uses the authentication service to login to the application. If the user is already logged in they are automatically redirected to the home page.
+
+mainModule:The Main module where the User enters after login under mainModule we have homeComponent.
+
+homeComponent: The home page shows the content what is exactly going into the project
+
+adapters: In the Project adapter is a library or package that allows Angular to work with other external libraries or frameworks. Adapters help to bridge the gap between Angular and these external tools, making it easier to integrate them into an project.In project we are using logged-in-user-adapter,login-adapter
+
+decorator:In this Project decorator is a TypeScript feature used to modify or extend the behavior of a class, method, property, or parameter. Decorators in Angular are typically used to add metadata to classes or their members, which is then used by Angular's dependency injection system, routing, or other Angular features.
```

Comment:
No need to include the definitions instead concentrate more on what is provided.
We can just give reference links for the definitions.